src/_data/toc/installation-guide.yml

```diff
@@ -60,44 +60,36 @@ pages:
         - label: Get the Magento software
           url: /install-gde/bk-install-guide.html
 
-        - label: (Contributor) Clone the Magento repository
-          url: /install-gde/prereq/dev_install.html
-          children:
-
-            - label: Update installation dependencies
-              url: /install-gde/install/prepare-install.html
-
-    - label: Quick start (recommended installation)
+    - label: Quick start install (non-contributor)
       url: /install-gde/composer.html
 
-    - label: Install options
-      url: /install-gde/bk-install-guide.html
-
-    - label: Command Line Installation
+    - label: Command Line install (contributor)
```

Comment:
This change conflates the contributor install method (git-based) with the only other method of installing Magento, which is the CLI. I can see this leading to some confusion later. 

Basically, in 2.4, the CLI is your ONLY option for installing Magento. The only difference between a contributor and non-contributor is that the former uses a **git-based** install method whereas the latter uses a **Composer-based** install method. 

Technically non-contributors could use an **archive-based** install method, but Magento recommends using Composer as the official method. We decided to de-emphasize the archive-based method in previous iterations of refreshing the Install Guide.

I envision something like this:

- Quick start install (non-contributor, composer-based)
- Quick start install (contributor, git-based)
- Advanced install (all other existing command line install topics)

I also think we need to take a closer look at whether the following **contributor** topics belong in the Install Guide at all:

- Update the Magento application (arguably belongs in the Software Update Guide)
- Add or update components (arguably belongs in the Software Update Guide)
- Change to a released version (i think should stay; it's still git-based install)
- Reinstall the Magento software (i think this topic should be removed entirely)

Reply:
OK. After break, I am thoroughly confused about where I was BUT I think I have a good start.

The items you mentioned above I have not yet done are:
* Move these topics (we'll discuss in next meeting): Update the Magento application, Add or update components, Reinstall the Magento software---However, I did remove them from the TOC so they do not appear in the install flow.
* Add content from the child pages to the top level, newly-created Quick start install (contributor) topic. I want to review my preview build with the team to ensure I'm on the right track before smoothing everything out.

**Preview build #137**

Reply:
@shrielenee, looks good! My suggestions for topic titles aren't very good, so hopefully the team will have better suggestions when we review next week, but you definitely captured the gist of my feedback. Thanks!